.changeset/wild-sheep-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Enhanced userPresence options by adding hours and minutes fields alongside seconds for idleTimeLimit, allowing more flexible idle time configuration.
```

Comment:
This changeset is too technical, the user does not know about the internal idleTimeLimit